pkg/config/config.go

```diff
@@ -37,6 +40,19 @@ type Secret struct {
 	Sinks  sink.Sinks    `yaml:"sinks"`
 }
 
+type HerokuEnv struct {
+	Bearer_Token string
+}
+
+func loadHerokuEnv() (*HerokuEnv, error) {
+	env := &HerokuEnv{}
+	err := envconfig.Process("heroku", env)
+	if err != nil {
+		return env, errors.Wrap(err, "Unable to load all the heroku environment variables")
+	}
+	return env, nil
```

Comment:
can simplify this as 
```suggestion
		return env, errors.Wrap(err, "Unable to load all the heroku environment variables")
```
since errors.wrap already does that nil check